readthedocs/core/static-src/core/js/doc-embed/sponsorship.js

```diff
@@ -106,9 +106,23 @@ Promo.prototype.post_promo_display = function () {
         // Alabaster only
         $('<hr />').insertBefore('#' + this.div_id + '.ethical-alabaster .ethical-footer');
     }
-
 };
 
+function adblock_admonition() {
+    console.log('---------------------------------------------------------------------------------------');
+    console.log('Read the Docs hosts documentation for tens of thousands of open source projects.');
+    console.log('We fund our development (we are open source) and operations through advertising.');
+
+    console.log('We promise to:');
+    console.log(' - never let advertisers run 3rd party JavaScript');
+    console.log(' - never sell user data to advertisers or other 3rd parties');
+    console.log(' - only show advertisements of interest to developers');
+    console.log('You can read more about our approach to advertising here: https://docs.readthedocs.io/en/latest/ethical-advertising.html');
+
+    console.log('%cPlease whitelist *.readthedocs.io on your adblocker!', 'font-size: 2em');
```

Comment:
I think we need to say `*.readthedocs.org` maybe?

Look at this

![captura de pantalla_2018-04-04_12-07-37](https://user-images.githubusercontent.com/244656/38322853-37ec2c14-3801-11e8-9ea8-b94481ccc807.png)

Reply:
it comes from EasyList

Reply:
That's a good catch. I need to review this. We may need both...

Reply:
This is a larger problem than I thought.

Whitelisting `*.readthedocs.io` is all that is necessary (on AdblockPlus at least) to allow Read the Docs ads on all of our sites hosted on a subdomain. However, ads will continue to be blocked on any custom domains where we host ads. Whitelisting `readthedocs.org` does not change this. This makes simple whitelisting impossible across all ~10,000 domains where we host ads.

Reply:
After reviewing their filter syntax, we need the following two filters:

* `@@||readthedocs.org/*` (exception for making an ad request)
* `@@readthedocs.io^$document` (exception for hiding elements)